tests/test_mmd.py

```diff
@@ -14,13 +14,32 @@
 
 def test_cifar_mmd_same():
     dmpair = DMPair(metric_config=METRIC_CONFIG)
+    dmpair.A.setup()
+    dmpair.B.setup()
```

Comment:
We could possibly move both `.setup()` calls inside of `dmpair.compute_similarity` (maybe with an appropriate warning/info) to avoid constantly writing it out (possibly with an argument defaulting to `True`?). This should be fine given that we'll be running two different pipelines for the model + attack training and the metrics

Reply:
yes, good idea. I'll put it in to check if already run and do this if not

Reply:
in fact, we don't actually need to call `setup()` at this point as we haven't implemented any transforms... I've removed this and linked to issue-15 in the code.